### PR TITLE
Feat(#16) - Improve the save task button

### PR DESCRIPTION
**Pull request by meetkparmar**
Feat #16

### CODE REVIEW

File: app/src/main/res/layout/dialog.xml [hunk 1]
```diff
@@ -44,7 +45,7 @@
       />
   <TextView
```
Comment (prudhvir3ddy) [inline app/src/main/res/layout/dialog.xml]: this should be a  button ?

**Comment by prudhvir3ddy** [comment]
@meetkparmar is it ready to merge ?

File: app/src/main/res/layout/dialog.xml [hunk 2]
```diff
@@ -42,9 +43,9 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent"
       />
-  <TextView
+  <Button
       android:id="@+id/save_tv"
```
Comment (prudhvir3ddy) [inline app/src/main/res/layout/dialog.xml]: update this save_tv to save_btn

File: app/src/main/java/com/prudhvir3ddy/todo_app_gettingthingsdone/view/main/BottomSheetDialog.kt [hunk 3]
```diff
@@ -35,6 +40,16 @@ class BottomSheetDialog : BottomSheetDialogFragment() {
     return v
   }
 
+  private fun setOnEditorAction(v: View) {
+    v.task_desc_et.setOnEditorActionListener { v, actionId, event ->
+      if (event != null && event.keyCode === KeyEvent.KEYCODE_ENTER || actionId == EditorInfo.IME_ACTION_DONE) {
+        save_tv.performClick()
```
Comment (prudhvir3ddy) [inline app/src/main/java/com/prudhvir3ddy/todo_app_gettingthingsdone/view/main/BottomSheetDialog.kt]: same here and other places

File: app/src/main/res/layout/dialog.xml (hunk 2 above)
Comment (meetkparmar) [reply app/src/main/res/layout/dialog.xml]: Ok

**Pull request merged**